ext/vscode/.vscode/launch.json

```diff
@@ -19,23 +19,30 @@
                 "DEBUGTELEMETRY": "true"
             },
             "outFiles": [
-                "${workspaceFolder}/dist/**/*.js"
-            ]
+                "${workspaceFolder}/dist/**/*.js",
+                "${workspaceFolder}/*.js"
+            ],
         },
         {
             "name": "Run AZD + Azure Resources Extensions",
             "type": "extensionHost",
             "request": "launch",
             "args": [
-                "--extensionDevelopmentPath=${workspaceFolder}/../../../../vscode-azureresourcegroups",
```

Comment:
If the AZD repo and Azure Resource Group ext repos are cloned side-by-side, and the "workspace folder" is the AZD VS Code ext folder, the Azure RG ext repo is 3 folders up, not 4 folders. I think this is most likely setup for someone working in this codebase.